webapp/components/PageFooter/PageFooter.vue

```diff
@@ -2,13 +2,21 @@
   <div id="footer" class="ds-footer">
     <a href="https://human-connection.org" target="_blank" v-html="$t('site.made')"></a>
     <span>-</span>
-    <nuxt-link to="/imprint">{{ $t('site.imprint') }}</nuxt-link>
+    <a
+      href="https://human-connection.org/impressum/"
+      target="_blank"
+      v-html="$t('site.imprint')"
+    ></a>
```

Comment:
What's the reason to remove the imprint, too? Keep in mind that at some point we have multiple instances of human connection. On that day, both data privacy section and imprint must be part of the code anyways. Pointing the data privacy to https://human-connection.org/en/privacy/ is a temporary workaround so it's easier for @datenbrei to keep it updated. Does the imprint also needs to be updated that often? Or can we just keep it?

Reply:
if we create an area in the admin area where these files can be changed.